app/templates/skeleton/modules/custom/skeleton_restful/plugins/restful/user/SkeletonAvailableResource.class.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * @file
+ * Contains SkeletonAvailableResource.
+ */
+
+class SkeletonAvailableResource extends \RestfulBase implements RestfulDataProviderInterface {
```

Comment:
move to one single resource `/api/users-availability/?email=foo@bar.com&name=test`